src/app/common/styles/sidebar.scss

```diff
@@ -44,20 +45,35 @@
   width: rem(18px);
   background-repeat: no-repeat;
   background-position: center;
+  box-sizing: border-box;
+  height: rem(14.45px);
+  width: rem(24.51px);
+
   &.dashboard {
-    color: #FFFFFF;
     background-image: url(../../../images/dashboard.svg);
   }
+
   &.istelle {
+    border: rem(1.5px) solid $lightAccent;
```

Comment:
Let's use a Mixin since all the properties for the society logo selector are the same.
We can avoid writing the same code over and over again.